v1/util/cloud_events_util.h

```diff
@@ -0,0 +1,35 @@
+#ifndef CLOUDEVENTSCPPSDK_V1_UTIL_CLOUDEVENTSUTIL
+#define CLOUDEVENTSCPPSDK_V1_UTIL_CLOUDEVENTSUTIL
+
+#include "third_party/statusor/statusor.h"
+#include "third_party/base64/base64.h"
+#include "proto/cloud_event.pb.h"
+
+namespace cloudevents {
+namespace util {
```

Comment:
Avoid collisions between nested namespaces and well-known top-level namespaces.

https://google.github.io/styleguide/cppguide.html#Namespace_Names has a example about util namespaces.

Reply:
Does the namespace cloudevents::cloudevents_util sound good? :)

Reply:
Should i change the top level namespace everywhere to be a bit more verbose? (ex: cloudevents_cpp_sdk) 
Or is that unecessary/ too long?

Reply:
Or it could be binder_util or parser_util depending on whatever it's used for. 

I wouldn't recommend changing it everywhere. Only change the names that are common like 'util'.

Reply:
done!